libp2p/multiaddress.nim

```diff
@@ -834,7 +834,7 @@ proc `&=`*(m1: var MultiAddress, m2: MultiAddress) =
   if not m1.validate():
     raise newException(MultiAddressError, "Incorrect MultiAddress!")
 
-proc isWire*(ma: MultiAddress): bool =
+proc isWire*(ma: MultiAddress): bool {.raises: [].} = # FIXME except but raises nothing
```

Comment:
@cheatfate when you have a minute can you check this?
It's using a pure `except:` to write the result but raises nothing apparently.